bank/html-podstrony/bilety/oferta-biletow.html

```diff
@@ -11,54 +11,93 @@
     <hr>
     <div id="choice-mpk" class="choice">
         <div class="ticket-list">
-            <div class="ticket-wrapper">
-                <button id="MPK-U1" class="ticket"></button>
-                <span class="ticket-text"> MPK - ulgowy, 1 miesiąc </span>
-            </div>
-            <div class="ticket-wrapper">
-                <button id="MPK-U2" class="ticket"></button>
-                <span class="ticket-text"> MPK - ulgowy, 2 miesiące </span>
-            </div>
-            <div class="ticket-wrapper">
-                <button id="MPK-U3" class="ticket"></button>
-                <span class="ticket-text"> MPK - ulgowy, 3 miesiące </span>
-            </div>
-            <div class="ticket-wrapper">
-                <button id="MPK-N1" class="ticket"></button>
-                <span class="ticket-text"> MPK - normalny, 1 miesiąc </span>
-            </div>
-            <div class="ticket-wrapper">
-                <button id="MPK-N2" class="ticket"></button>
-                <span class="ticket-text"> MPK - normalny, 2 miesiące </span>
-            </div>
-            <div class="ticket-wrapper">
-                <button id="MPK-N3" class="ticket"></button>
-                <span class="ticket-text"> MPK - normalny, 3 miesiące </span>
-            </div>
+            <button class="ticket-wrapper">
+                <div id="MPK-U1" class="ticket">
```

Comment:
znacznik div w znaczniku button nie jest zgodny ze standardem html -> zmienić na span (we wszystkich button class="ticket-wrapper")